packages/dds/tree/src/test/feature-libraries/chunked-forest/codec/compressedEncode.spec.ts

```diff
@@ -130,7 +138,7 @@ describe("compressedEncode", () => {
 				const processed = handleShapesAndIdentifiers(version, [buffer]);
 				assert(processed.data.length === 1);
 				const stream = { data: processed.data[0], offset: 0 };
-				const decoded = readValue(stream, shape);
+				const decoded = readValue(stream, shape, testIdCompressor);
```

Comment:
I've added some unit tests in `schemaBasedEncoding.spec.ts` using identifiers, but since the `compressedEncode.spec.ts` tests do not take in schema, I'm not sure how I could write tests that specify that they are identifiers.

Would the tests use `StableId` as string values without specifying the schema for these tests?